mailer.rst

```diff
@@ -1266,6 +1266,10 @@ The following transports only support tags:
 
 * OhMySMTP
 
+The following transports only support metadata:
+
+* Amazon SES
```

Comment:
The Amazon platform refers to this data as *tags* but since they contain a key and a value, we call this metadata. Should I add a note to explain this?

Reply:
Kevin, I think you are right that we should add a note to avoid confusion. I did that while merging: https://github.com/symfony/symfony-docs/commit/f857b22e1043c50e887adbbcf45470fec7f2ee10

Thanks!